plugins/vli/vli-lenovo.quirk

```diff
@@ -258,6 +258,29 @@ ParentGuid = USB\VID_17EF&PID_A392
 Plugin = vli
 GType = FuVliUsbhubDevice
 
+#Lenovo Test Device 2023
```

Comment:
100% a nit, but space after the # please

Reply:
No problem! I'm actually waiting on an official name for the device, please stand by.